### PR TITLE
Remove padding when text is null

### PR DESCRIPTION
**Pull request by Benedito0**
| - | - |
| --- | --- |
| Decisions |The need for this came from working on this [issue](https://github.com/ripe-tech/ripe-pulse/issues/146) on Pulse, noticing the disalignment of icon in the button. Keeping the padding when there is a slot because most usages of this component use the slot with just text. |

### CODE REVIEW

**Review by gcandal — APPROVED** [review]

File: vue/components/ui/atoms/button-color/button-color.vue [hunk 1]
```diff
@@ -406,6 +407,11 @@ export const ButtonColor = {
             }
```
Comment (joamag) [inline vue/components/ui/atoms/button-color/button-color.vue:408]: add a `no-text` class for when there's no text and use it to invalidate the padding

**Review by joamag — CHANGES_REQUESTED** [review]

**Review by joamag — APPROVED** [review]

**Review by joamag — APPROVED** [review]

**Comment by joamag** [comment]
@gcandal I believe this requires back-port

**Comment by gcandal** [comment]
> @gcandal I believe this requires back-port

@Benedito0 please take care of this.

**Comment by joamag** [comment]
@gcandal it seems that there's no need for back-porting

**Pull request merged**